boa/src/builtins/number/mod.rs

```diff
@@ -428,9 +425,9 @@ impl Number {
         number.set_field("MIN_SAFE_INTEGER", Value::from(-9_007_199_254_740_991_f64));
         number.set_field("MAX_VALUE", Value::from(std::f64::MAX));
         number.set_field("MIN_VALUE", Value::from(std::f64::MIN));
-        number.set_field("NEGATIVE_INFINITY", Value::from(f64::NEG_INFINITY));
-        number.set_field("POSITIVE_INFINITY", Value::from(f64::INFINITY));
-        number.set_field("NaN", Value::from(f64::NAN));
+        number.set_field("NEGATIVE_INFINITY", Value::from(std::f64::NEG_INFINITY));
+        number.set_field("POSITIVE_INFINITY", Value::from(std::f64::INFINITY));
+        number.set_field("NaN", Value::from(std::f64::NAN));
```

Comment:
Why was this changed? I think we should use the built-in constants instead of the module-level constants. They were stabilized in 1.42 or 1.43 and the others are more or less deprecated in idiomatic code.